### ISSUE TITLE
Type information display wrong and no completions for sys.exc_info with latest typeshed.

### ISSUE THREAD

**Issue by linette-zyy** [microsoft/pylance-release#1690]
**Environment**
![image](https://user-images.githubusercontent.com/59192491/129868202-404f157b-8fd4-4745-a40b-1d41386e408e.png)

**Steps to Reproduce**
1. Create a python application and input below
```
import sys
e1, e2, e3 = sys.exc_info()
```
2. Move the cursor to e1, e2, e3
3. And then custom Typeshed path
  a. git clone https://github.com/python/typeshed, enter the local path on disk where you cloned the repository
  b. open typeshed/stdlib/sys.pyi and replace
		def exc_info() -> _OptExcInfo 
		 
		With this instead:
		 
		def exc_info() -> Tuple[Optional[Type[BaseException]],
		                        Optional[BaseException],
		                        Optional[_version_info]]: ...

  c. Close and reopen the project
  d. It should now report e3 as being a version info instance

**Expected behavior**
Before custom Typeshed path
![image](https://user-images.githubusercontent.com/59192491/129868280-e465de8a-5318-4e2d-a7a4-a0a4c9893c0c.png)
After custom Typeshed path
![image](https://user-images.githubusercontent.com/59192491/129869482-25b4c711-b200-4a6a-a480-f91f13cd42dd.png)

**Actual result**
Before custom Typeshed path
![image](https://user-images.githubusercontent.com/59192491/129868739-7721cf16-85e7-49f2-8261-44331d82f7ff.png)
![image](https://user-images.githubusercontent.com/59192491/129868767-e26d98b3-903c-45f4-b1e6-16da0da794c7.png)
![image](https://user-images.githubusercontent.com/59192491/129868790-0d215f74-4315-47b5-8ab2-6bee2ff310dc.png)
After custom Typeshed path
![image](https://user-images.githubusercontent.com/59192491/129869767-0344c949-4e50-436b-9be4-6bd4eee6fd8c.png)
![image](https://user-images.githubusercontent.com/59192491/129869743-df731e2d-2488-4ad3-89a4-af240545b8ab.png)

**Comment by rchiodo** [comment]
I assume this is a file watcher problem? That reopening the solution would cause the correct types to be loaded?

**Comment by jakebailey** [comment]
Can we get some trace logs to verify that the right typeshed is being used?

I don't think that we watch the typeshed path; the default one is intended to be static, and I don't think we have any special file watching for custom typeshed paths. A reload might fix this, if typeshed is being edited while the editor is open.

**Comment by linette-zyy** [comment]
Edit the sys.ini 
![image](https://user-images.githubusercontent.com/59192491/129992786-666f7dfc-56bf-4ba1-ad1e-e9529710fd1d.png)
I custom Typeshed path like this
![image](https://user-images.githubusercontent.com/59192491/129992744-1c916766-449c-431a-84c4-3da874895154.png)
And restart the VS, it was no different from before editing.
![image](https://user-images.githubusercontent.com/59192491/129992974-6ca4c692-58b8-443e-b4c1-f1969fe1525c.png)
![image](https://user-images.githubusercontent.com/59192491/129992996-6e428d52-c3e1-4220-b1d3-2502cdbaa8ea.png)
And I get an error message on Pylance output window
`Error: Warning: Info: Info: Info: Info: Info: Error reading settings: TypeError: Cannot read property '0' of nullstubPath typings is not a valid directory.Assuming Python platform WindowsSearching for source filesNo source files found.Background analysis(1) root directory: c:\program files\microsoft visual studio\2022\preview\common7\ide\extensions\microsoft\python\core\pylance\distBackground analysis(1) started`

**Comment by jakebailey** [comment]
Yeah, this implies that PTVS is setting the stub path to `null` for some reason, when it should be a path, and that's causing typeshed to fail to be loaded.

**Comment by linette-zyy** [comment]
Yes, it fixed on the latest build.
![image](https://user-images.githubusercontent.com/59192491/131821728-03d08071-e688-4215-a7c3-ddd6f07751ac.png)
![image](https://user-images.githubusercontent.com/59192491/131821749-3496afb3-34d1-4d52-bf27-07f2eefade8e.png)
![image](https://user-images.githubusercontent.com/59192491/131821776-a9e69f06-59e0-4acf-8492-3b92a72d6786.png)
